src/settings.h

```diff
@@ -50,6 +51,22 @@ class CSettings : public QObject
         strFileName ( "" )
     {
         QObject::connect ( QCoreApplication::instance(), &QCoreApplication::aboutToQuit, this, &CSettings::OnAboutToQuit );
+
+#ifndef HEADLESS
+        const QGuiApplication* pGApp = dynamic_cast<const QGuiApplication*> ( QCoreApplication::instance() );
```

Comment:
Maybe put a comment here describing what and why this is needed.

Reply:
Done.